### ISSUE TITLE
ERROR:Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by Lyxxx323** [HHTseng/video-classification#35]
Hello,When I was running UCF101_CRNN.py, I got this error:
`Process finished with exit code -1073741819 (0xC0000005)`
An error occurred while running to this line of code：
`loss.backward()`
So I am wondering if you could please kindly help me.

**Comment by abhiray92** [comment]
@Lyxxx323 This is an access issue, please check if you have necessary permissions in the folder you are trying to access. Moreover, please try to place your dataset/Model Paths on a different folder/drive.

**Comment by Lyxxx323** [comment]
Thank you so much！







At 2020-11-18 23:14:28, "Abhishek Ray" <notifications@github.com> wrote:

@Lyxxx323 This is an access issue, please check if you have necessary permissions in the folder you are trying to access. Moreover, please try to place your dataset/Model Paths on a different folder/drive.
